colossalai/amp/naive_amp/_fp16_optimizer.py

```diff
@@ -30,6 +42,8 @@ def _multi_tensor_copy_this_to_that(this, that, overflow_buf=None):
     if overflow_buf:
         overflow_buf.fill_(0)
         # Scaling with factor `1.0` is equivalent to copy.
+        global multi_tensor_scale
```

Comment:
where is multi_tensor_scale? do we need this line?

Reply:
Fixed the typo.